firebase-firestore/src/main/java/com/google/firebase/firestore/FirebaseFirestore.java

```diff
@@ -362,6 +362,16 @@ public WriteBatch batch() {
     return shutdownInternal();
   }
 
+  /**
+   * Wait until all pending writes existed at the time of calling are sent to the backend.
```

Comment:
A bunch of comments on this:
  * Phrase comments with an implicit "this method" at the beginning
  * missing "that" between "writes existed"
  * This doesn't just wait until writes are sent, but also waits for them to complete

> [This method ...] *Waits* until all pending writes that existed at the time of calling are have been successfully written to the server. The task will will not resolve while the client is offline.

Reply:
Done.